sdk/rpc/server/server.go

```diff
@@ -116,16 +117,25 @@ func (t *RPCServer) ExecutableNeedsAuth(req proto.ExecutableNeedsAuthRequest, re
 
 // CredentialImport is a remote version of the Import() function in schema.CredentialType.
 // The call is forwarded to the Import() function of the credential identified by req.CredentialID.
-func (t *RPCServer) CredentialImport(req proto.ImportCredentialRequest, resp *sdk.ImportOutput) error {
+func (t *RPCServer) CredentialImport(req proto.ImportCredentialRequest, resp proto.ImportCredentialResponse) error {
+	defer func() {
+		if err := recover(); err != nil {
+			resp.Panic = &proto.Panic{
+				Error: fmt.Sprintf("your locally built plugin's importing failed with the following panic: %s", err),
```

Comment:
The fact that we're dealing with an ImportRequest should be clear from the response type. Similarly, the fact that this is set in a struct called `Panic` should indicate that this is about a panic.

So if we just set `Error: err.Error()` we provide the most flexibility for the consumer (CLI) of this struct. The CLI can add the extra context if it sees fit or leave it out if it is not needed.

Reply:
Alright, makes sense. The only problem I still see here is that the `err`  we get from recover has type any, so it does not have any `.Error()` method, and casting to an error could sometimes fail because `recover()` does not always return an error. It was designed to also return the data-structure with which the panic has been initialized (`panic("oh- oh")`). Should we assume that in our case, we'll only catch panics from which we can recover errors, and cast?

Reply:
Oh, yeah, you're right! I keep on forgetting that `panic()` accepts any value 😅 

Let's not cast the value; that could trigger another panic. What we could do instead, is rename the `Error` field. If it is instead called something like `Recovered` or `RecoveredString`, it would make more sense to just set it to `fmt.Sprintf("%s", r)` (where `r` is `recover()`'s return value).

Reply:
Makes sense, thanks,done!